### DISCUSSION TITLE
Evaluating CoreUI for a laravel project - stuck at the first hurdle!

### DISCUSSION THREAD

**Discussion by jeremy0505** [coreui/coreui#385] [Q&A]
Hi I have searched high and low (presumably though not in the right places yet!!) for the "dummies guide" to getting a mock up / prototype running with CoreUI (free).

Using the info here https://coreui.io/docs/getting-started/introduction/ I assumed I could embed the two lines (one for CSS, one for scripts) and effectively then access all of the resources from the CDN. I added these lines

`<link href="https://cdn.jsdelivr.net/npm/@coreui/coreui@4.1.1/dist/css/coreui.min.css" rel="stylesheet" integrity="sha384-4RjTy+ihDlYQqAQJodVMv1QuiX0mRlR0xtuKKu1P8I2TxVdGPXaW+VxJ6PirwMm4" crossorigin="anonymous">`

and 

    `<script src="https://cdn.jsdelivr.net/npm/@coreui/coreui@4.1.1/dist/js/coreui.bundle.min.js" integrity="sha384-PXOm0LjM7kBY7at2HNqu2pAJsTVQUPRzpJSwQD0V//zsjFAcv61qjiZj3ELesLb1" crossorigin="anonymous"></script>`

Then I wanted to implement the (as it appears) "dummy" CoreUI admin template and then start to dissect it, see what I wanted to use / replace etc. so I copied some of the code (for the sidebar) but it just made a huge mess on my browser. Looking at dev tools I can see it's attempting to load resources e.g.

http://localhost:8000/assets/brand/coreui.svg

Which suggests to me that I need to have the files all local... 

I *suspect* that the HTML code on the demo admin template (https://coreui.io/demo/4.0/free/) is coded to load resources from the host running the page as opposed to being loaded from the CDN

So I guess a few questions:

1) what's the recommended way to "get started" with a CoreUI admin template?
2) can I run the entire thing (in prototype mode) without installing assets locally?
3) if so do I need to refer to assets in some other/special way?
4) if I *do* need the file locally, what exactly do I need to do? I am running with Laravel 9 and had assumed that 
  `$ composer require coreui/coreui`
  `$ npm install && npm run dev`
would take care of dropping everything into the right places.

Sorry to be such a noob but am just finding (trying to!) my feet.

cheers,
J

**Comment by 10JePoy28** [comment]
I tried following the installation procedure from here (https://github.com/coreui/coreui-free-bootstrap-admin-template) and it worked,
on your laravel project folder git clone or download the zip file and extract the contents on a folder inside your laravel project, I usually call my folder "codeui" after that go to terminal->go to "codeui" directory-> run `npm install` -> run `npm start`
